cmd/common/image/inspect.go

```diff
@@ -162,5 +156,15 @@ func NewInspectCmd(runtime runtime.Runtime) *cobra.Command {
 		fmt.Sprintf("Output mode, possible values are, %s", strings.Join(outputModes, ", ")),
 	)
 
+	var skipParams []string
+	if commandMode == common.CommandModeAttach {
+		skipParams = append(skipParams, "!attach")
+	}
```

Comment:
We don't want to add the headless-related flags to the inspect command as it will never run in headless mode. So, we don't need to pass the `commandMode` around. We can simply define the `skipParams` like this:

```golang
        // We don't want to add the headless-related flags to the inspect command
	skipParams := []string{"!attach"}
```